scripts/pip-install-test.sh

```diff
@@ -0,0 +1 @@
+pip install pytest
```

Comment:
Can't we just have a requirements-dev.txt or use poetry or something to manage these